src/components/Pagination/Pagination.tsx

```diff
@@ -124,7 +140,7 @@ const Pagination = (props: PaginationProps) => {
       : [];
     if (!selectedPageSize) {
       setSelectedPageSize(
-        value.limit ? value.limit : po.length > 0 ? po[0].value : 10
+        value.limit ? value.limit : po.length > 0 ? po[0].value : 1800
```

Comment:
Please move 1800 and any other hardcoded constants in code to a constants file per component. This will make it easier to understand and maintain values.